app/serializers/utility_serializer.rb

```diff
@@ -5,6 +5,7 @@ def self.make_json(data)
 		{
 			'data': data[:businesses].map do |biz|
 				{
+					'id': biz[:id],
```

Comment:
Thanks for adding this - we'll adjust things on FE as well

Reply:
Thank you!